src/components/SchedulePanel/components/OfficeHoursDialog/index.js

```diff
@@ -0,0 +1,526 @@
+import React, { Fragment } from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import FormGroup from '@material-ui/core/FormGroup';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { gordonColors } from '../../../../theme';
+import Checkbox from '@material-ui/core/Checkbox';
+import myschedule from './../../../../services/myschedule'
+
+import './officehoursdialog.css';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+export default class HoursDialog extends React.Component {
```

Comment:
Can we rename this component, since it is no longer specific to OfficeHours, but handles all custom events?